cluster/addons/calico-policy-controller/MAINTAINERS.md

```diff
@@ -0,0 +1,6 @@
+# Maintainers
```

Comment:
I based this on the dashboard and node-problem-detector folders adjacent to this.  I'm happy to change it: just pointing out that the closest equivalents currently look like this.

Which would you prefer?